### PR TITLE
Ft/restful api/151208540

### PR DESCRIPTION
**Pull request by johadi**
#### What does this PR do?
It extends Meteor DDP methods in this project by providing it with RESTful API.
#### Description of Task to be completed?
When a user makes a request to a valid RESTful API endpoint , then a response in the format of JSON should be sent to the user.
#### How should this be manually tested?
Using POSTMAN
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
#151208540
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: server/startup/restful_api/api.js [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                message: "Collection is deleted",
+                data: updatedCollection
+              };
+            }
+          }
+        }
+
+        // patch: {
+        //   // TODO: write the patch action() method
+        // }
```
Comment (mradeybee) [inline server/startup/restful_api/api.js]: Remove commented (dead) code.

**Review by mradeybee — CHANGES_REQUESTED** [review]
A little comment and I cant see any test.

File: server/startup/restful_api/api.js (hunk 1 above)
Comment (mradeybee) [reply server/startup/restful_api/api.js]: @johadi10 address this and let me know.

Comment (johadi) [reply server/startup/restful_api/api.js]: @mradeybee Please review. I have removed the dead code

**Review by mradeybee — APPROVED** [review]
LGTM

**Pull request merged**